shared/src/commonMain/kotlin/io/customer/shared/util/DateTimeUtil.kt

```diff
@@ -0,0 +1,23 @@
+package io.customer.shared.util
+
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+
+internal interface DateTimeUtil {
```

Comment:
I don't see this code added anywhere. Reason for adding it now in this PR?

Reply:
We'll need it when adding tasks to queue. Added it to this PR as it is almost replica of our current implementation.